software-modules/internet/chromium/chromium.sh

```diff
@@ -20,12 +20,32 @@ TARGET_DIR="/run/initramfs/memory/system/huronOS/software/internet/"
 
 ## Install software
 apt update
-apt install --yes --no-install-recommends $NAME
+apt install --yes --no-install-recommends $NAME python-is-python3
 apt autoremove --yes
 
 ## Prepare final files
 cp ./$NAME.desktop /usr/share/applications/
 
+## Prepare bookmarks extension
+# The extension
+curl -o /usr/share/bookmark-bridge.crx -L https://github.com/huronOS/bookmark-bridge/releases/latest/download/bookmark-bridge.crx
+
+# The extension loader
+CHROMIUM_EXT_LOADER_PATH="/usr/share/chromium/extensions"
+mkdir -p $CHROMIUM_EXT_LOADER_PATH
+curl -o "$CHROMIUM_EXT_LOADER_PATH/cbfajpicdmpnnoijdmhdomaailmdaiim.json" https://raw.githubusercontent.com/huronOS/bookmark-bridge/development/chrome-extension-loader.json
+
+# The extension's native host
+CHROMIUM_NATIVE_HOST="/etc/chromium/native-messaging-hosts"
+mkdir -p $CHROMIUM_NATIVE_HOST
+curl -o "$CHROMIUM_NATIVE_HOST/bookmarks_fetcher.json" https://raw.githubusercontent.com/huronOS/bookmark-bridge/development/chrome-native-host.json
+
+# The middleman between huron and the browser
+curl -o /usr/bin/bookmarks-fetcher.py https://raw.githubusercontent.com/huronOS/bookmark-bridge/development/app/bookmarks-fetcher.py
+curl -o /usr/bin/current-bookmarks https://raw.githubusercontent.com/huronOS/bookmark-bridge/development/app/current-bookmarks
+chmod +x /usr/bin/bookmarks-fetcher.py
```

Comment:
Best to specify all permissions (like 744) instead of +x, +x does enable execution for everyone. Explicit I would say that it's better.

Reply:
Okay, got it